app/styles/users.scss

```diff
@@ -0,0 +1,18 @@
+.resultslist-header {
```

Comment:
this is going to override resultslist-header for every list on the site.  It needs to be underneath another class or the name needs to change.

File is called users.scss so I would ecpect to find a .users {

File should be in components.  Not in styles.